<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expense Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>471</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Expense Tracker</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>200</y>
          <width>511</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="button_add_expense">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Add new expense</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_show_statistics">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Show statistics</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_reset">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Reset expenses</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>421</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add new expense</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>230</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cost</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>310</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>400</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Details</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_button">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>550</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancel_button">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>550</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="details_text_edit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>420</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="category_combo_box">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>170</y>
          <width>291</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Food</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Transportation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medical &amp; Healthcare</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clothes &amp; shoes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Entertainment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Housing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>320</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>250</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>411</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Statistics</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Total Sum</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_total_sum">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>130</y>
          <width>131</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_cancel">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>540</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_categories_charts">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>280</y>
          <width>311</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Categories charts</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="button_cancel_categories_charts">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>560</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_categories_chart">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>651</width>
          <height>501</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>categories_chart.jpg</pixmap>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
